### PR TITLE
added lesson02 assignment

### PR DESCRIPTION
**Pull request by mluckeroth**
grid printer exercise submission.

### CODE REVIEW

**Comment by mluckeroth** [comment]
add FizzBuzz submission

**Comment by mluckeroth** [comment]
added series submission

**Comment by natasha-aleksandrova** [comment]
@mluckeroth did you mean to close it?

**Comment by mluckeroth** [comment]
no, I'm not sure what to do.

**Comment by natasha-aleksandrova** [comment]
Might have accidentally pressed the button. There is a button at the bottom with text "Reopen Pull Request"

I will go ahead and reopen it.

**Comment by mluckeroth** [comment]
thank you.  Is there anything else I need to do?

**Comment by natasha-aleksandrova** [comment]
No you're good 🙂 

I will review your code today.

File: students/mark_luckeroth/lesson02/FizzBuzz.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+def fizz_buzz(start=1, end=100):
```
Comment (natasha-aleksandrova) [inline students/mark_luckeroth/lesson02/FizzBuzz.py]: note that 100 is not inclusive so you would want up to 101

File: students/mark_luckeroth/lesson02/FizzBuzz.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    """prints numbers start to end, inclusive
+       replaces multiples of 3 with 'Fizz'
+       replaces multiples of 5 with 'Buzz'
+       replaces multiples of 15 with 'FizzBuzz'"""
+    for i in range(start,end):
+        if (i%3 == 0) and (i%5 == 0):
+            print('FizzBuzz')
+        elif (i%3 == 0):
```
Comment (natasha-aleksandrova) [inline students/mark_luckeroth/lesson02/FizzBuzz.py]: no need for parens, they are redundant.

File: students/mark_luckeroth/lesson02/series.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        return one
+    else:
+        return sum_series(n-2,zero,one) + sum_series(n-1,zero,one)
+
+#Test block
+
+assert fibonacci(9) == 34         #test verifies fibonacci function
+assert lucas(5) == 11             #test verifies lucas function
+assert sum_series(8) == 21        #test verifies sum_series function with arg[1] and arg[2] unspecified
+assert sum_series(8,2,6) == 152   #test verifies sum_series function with all arguments specified
```
Comment (natasha-aleksandrova) [inline students/mark_luckeroth/lesson02/series.py]: 👍

**Pull request merged**